### DISCUSSION TITLE
Can we get some feedback on the 0.7.0 release schedule?

### DISCUSSION THREAD

**Discussion by David-Else** [zealdocs/zeal#1308] [General]
The last release was at the end of 2018, `0.7.0` seems to have been on the verge of release for a very long time now and the [milestone](https://github.com/zealdocs/zeal/milestone/14) says it is 97% complete.

It would be really helpful to get an idea of when the new version might come out to help plan the new year's development setup. Thanks for all the hard work, and I hope you don't think it is rude to ask, not trying to rush you. Cheers!

**Comment by trollixx** [comment]
v0.7.0 is way overdue, that's true. Migration to Qt WebEngine and my lack of time delayed it even further. Currently there are still numerous regressions, but no major blockers, and overall I think the code is in a decent shape. Fixing AppImage packaging is not a hard requirement for v0.7.0 either. So currently I'd like to cut the release in the coming weeks but it depends on the available spare time. I also need to do some more testing and definitely fix the CI.

 In the meanwhile, you can build the most recent code and give it a try.

**Reply by spencertweed** [reply]
Just wanted to let you know I've been avidly using Zeal for over a year now and I'm very excited to see a new release when you get around to it!

**Reply by swsnr** [reply]
@trollixx Would you consider making an alpha or beta release?  If only to show that some progress is happing?  Right now it says that the last release was in 2018 on the frontpage :slightly_smiling_face:

**Reply by trollixx** [reply]
Two years later we finally have [binary CI builds](https://github.com/zealdocs/zeal/actions/workflows/build-check.yml) for Windows and Linux (AppImage). They are attached as artifacts to each build. Please give them a try and report back any issues.

I have a short list of packaging related items that require addressing prior to tagging a new release but otherwise I think v0.7.0 will be out in no time.

**Comment by ezxpro** [comment]
@trollixx Any updates on this?

**Reply by slashsbin** [reply]
any updates or even a beta release would be appreciated.

**Reply by trollixx** [reply]
See https://github.com/zealdocs/zeal/discussions/1308#discussioncomment-5565548.

**Comment by ghost** [comment]
It would be great if there was an update, because I have seen some details in the visualization of some Docsets (It doesn't load the styles).
Greetings

**Reply by ezxpro** [reply]
It seems Zeal Docs has basically been abandoned. There are some occasional commits, but no ongoing development is present.

I think it deserves a fork, it's a good project and what the developers have done already is amazing, and I'm thankful for that.

Unfortunately, I don't have the skills necessary to maintain a project like this, I'd very much like to fork it and maintain it myself, but I'd have to improve my programming skills and learn C++ first.

**Reply by trollixx** [reply]
Not intentionally abandoned but as it often happens the real life has been getting in the way.

Unfortunately, nobody with a required skill set ever volunteered or stuck around long enough to become a maintainer. I would want to see a steady stream level of engagement before granting more access, but otherwise I'd love to have more people contributing to Zeal. It gets lonely working on it by myself.

Forking is also completely fine by me, that's what open source is for. A fork would need to procure their own permission to use Dash docsets and come up with an alternative to Zeal API/redirect servers which I am paying for.

**Reply by ernstki** [reply]
@trollixx Thanks for doing the lonely work. It doesn't go unnoticed.

To the others, Zeal's API/redirect servers—which have worked flawlessly for me personally, through years of use—aren't free. Maybe see #1490 if you'd like to put some money in the hat and help out, or just say thank you with something besides a thumbs-up emoji.

**Reply by CuteWaterBeary** [reply]
I tried to follow your build instructions on windows but they led to no success a few months back. If you could update the build instructions to make sure all the steps and links to anything else we need to download are there then we could actually help out on this project.

**Comment by edgimar** [comment]
Now that https://github.com/zealdocs/zeal/commit/a5247351295a971252346976fbbf0524ee2754cb (better dark-mode on Windows) is committed, it would be great to have a new release!

**Comment by trollixx** [comment]
[v0.7.0](https://github.com/zealdocs/zeal/releases/tag/v0.7.0) is out!
